packages/amplify-appsync-simulator/src/velocity/value-mapper/map.ts

```diff
@@ -56,7 +57,7 @@ export class JavaMap {
   }
 
   keySet() {
-    return new JavaArray(Array.from(this.map.keys()), this.mapper);
+    return map(Array.from(this.map.keys()));
```

Comment:
Please update this to not use the default `map` function.

Reply:
done